articles/client-platforms/angular2/06-rules.md

```diff
@@ -33,4 +32,4 @@ Then, display user profile's new `country` attribute added in the created rule a
 ![Country rule sample](/media/articles/angularjs2/rule-country-show.png)
```

Comment:
Image is broken https://auth0-docs-content-pr-1763.herokuapp.com/docs/quickstart/spa/angular2/06-rules#test-rule-result